rasa_core/channels/rasa_chat.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+
+import requests
+from flask import Blueprint, request, jsonify, abort
+from flask_cors import CORS, cross_origin
+
+from rasa_core.channels.channel import UserMessage, OutputChannel
+from rasa_core.channels.rest import HttpInputComponent
+
+logger = logging.getLogger(__name__)
+
+
+class NoOutputChannel(OutputChannel):
```

Comment:
would be just use the `CollectingOutputChannel` for this? https://github.com/RasaHQ/rasa_core/blob/master/rasa_core/channels/direct.py#L4